src/main/kotlin/io/snyk/plugin/services/SnykCliAuthenticationService.kt

```diff
@@ -47,7 +48,7 @@ class SnykCliAuthenticationService(val project: Project) {
         downloadCliIfNeeded()
         if (getCliFile().exists()) executeAuthCommand()
         if (isAuthenticated) executeGetConfigApiCommand()
-
+        LanguageServerWrapper.getInstance().updateConfiguration()
```

Comment:
Question: Was this something we should have added before and you just noticed?

Reply:
It was a merge problem. The method was renamed...

Reply:
Ok, let me rephrase. I noticed, that the token was not updated in Language Server after authentication. So I added it.